### ISSUE TITLE
[Feature] add Heilbronn campus available rooms

### ISSUE THREAD

**Issue by mohanadkandil** [TUM-Dev/Campus-Backend#274]

**Comment by CommanderStorm** [comment]
Unshure if this can be solved:

The free rooms is provided by the iris ("Intelligentes Lernraum-Management")
See https://www.devapp.it.tum.de/iris/app/about and https://www.sv.tum.de/sv/das-machen-wir/projekte/lernraeume/

I am not aware such a system exists on the schwarz' side of TUM. Could you point me to such a system if it exists?

**Comment by mohanadkandil** [comment]
> Unshure if this can be solved:
> 
> The free rooms is provided by the iris ("Intelligentes Lernraum-Management") See https://www.devapp.it.tum.de/iris/app/about and https://www.sv.tum.de/sv/das-machen-wir/projekte/lernraeume/
> 
> I am not aware such a system exists on the schwarz' side of TUM. Could you point me to such a system if it exists?

Yeah it's not IRIS. Do you know any contact at TUM who's responsible for these things ?
maybe we can come up with any bridge-like script between data with Schwarz and IRIS (so they can sync together).

**Comment by CommanderStorm** [comment]
The contact for TUM can be found under the links above.

Could you give me a pointer to the system currently in use in Heilbronn?

**Comment by mohanadkandil** [comment]
> The contact for TUM can be found under the links above. (we regularly communicate with them)
> 
> Could you give me a pointer to the system currently in use in Heilbronn? (is there such a system? are there learning rooms?)
> 
> When I was there, I did not see such a system
> ![20221015_164101](https://user-images.githubusercontent.com/26258709/277332810-b5ab4b14-9172-48ca-a1f5-415ccf7519af.jpg)

I don't know the current system and I didn't see such a system for rooms etc in Heilbronn campus. Indeed, I'll investigate the current systems (if available) in different buildings and I'll update you asap.

**Comment by mohanadkandil** [comment]
Moreover, it's mentioned [here](https://wissensstadt.hn/schwarz-campus-service/) the TUM Campus Heilbronn is a member of the Schwarz Campus Service, which means that everything is operated by their Software
